src/js/common/components/Nav.jsx

```diff
@@ -56,13 +56,13 @@ class Nav extends React.Component {
               <div>
                 <h5>{chapter.name}</h5>
                 <ul className="usa-unstyled-list">
-                  {chapter.pages.map(page => {
+                  {chapter.pages.length > 1 ? chapter.pages.map(page => {
```

Comment:
This got lost at some point. We should hide the page from the side nav when there's only one page in a chapter.